web_src/js/components/ContextPopup.vue

```diff
@@ -16,6 +16,10 @@
         </div>
       </div>
     </div>
+    <div v-if="!loading && issue === null">
+      <p><small>Error occured</small></p>
```

Comment:
I'm not sure if this can be I18n'ed

Reply:
What we could potentially do is add another variable to `window.config`: `window.config.i18n`, where all dynamically shown strings are set in the currently selected locale.
I can think of two feasible approaches to do this:
As the first option, we declare a new prefix for resource strings that are sent to the frontend (i.e. `frontend.<x>` or `dynamic.<x>`) and then autogenerate the corresponding map for all strings that have this prefix.
Alternatively, we manually add the strings to generate the map to send to the frontend.

But either way, I think this is out of scope for this PR. Or what's your opinion on that?

Reply:
> : `window.config.i18n`,

Their already seems to be such thing 😅. I forget it existed. https://github.com/go-gitea/gitea/pull/18111/commits/3b87a0d015a28af2eeedabbcb4a7b7fd41773a22

Reply:
The funny thing is: I was also unsure if it existed or not because I remember having it seen somewhere, but a quick grep and browser console search revealed nothing. So I thought it was implemented in some PR without having been merged, but apparently I was wrong.